### PR TITLE
Allow to select DevExtreme version in the ThemeBuilder (T747071)

### PR DESCRIPTION
**Pull request by babich-a**
@nightskylark please look at the way we get the DevExtreme version of the application (getDevExtremeVersion).

devextreme-cli@1.1.5 with the suggested changes was published to the local npm registry.

### CODE REVIEW

File: commands/themebuider.js [hunk 1]
```diff
@@ -75,73 +80,115 @@ const readInput = options => new Promise(resolve => {
...
-                    content += formatKey + ': ' + metadata[metadataKey] + ';\n';
-                }
-            } else if(options.command === commands.BUILD_META) {
-                const metadata = getMeta(result.compiledMetadata, options.base, filter);
-                let exportedMeta = [];
+    try {
+        console.log(`The ${version} version is used.`);
+        await installThemeBuilder(version);
+    } catch(e) {
+        console.log(`The devextreme-themebuilder npm package of v${version} was not installed. Please verify you are using v${version} or higher and examine the installation error log to further troubleshoot the issue.`);
```
Comment (RomanTsukanov) [inline commands/themebuider.js]: > The devextreme-themebuilder npm package of **v${version}** was not installed. Please verify you are using **v${version}** or higher... 

I suppose these should be two different versions.

File: commands/commands.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "commands": [{
         "name": "new",
         "descripttion": "Create a new DevExtreme application",
-        "usage": "devexterme new <application-type> <name> [options]",
+        "usage": "devextreme new <application-type> <name> [options]",
         "arguments": [{
             "name": "name",
             "descripttion": "The application name"
```
Comment (BingoRUS) [inline commands/commands.json]: If you started dealing with typos, let's fix another one :)
```suggestion
            "description": "The application name"
```

Comment (babich-a) [reply commands/commands.json]: done

File: commands/themebuider.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
-const themeBuilder = require('devextreme-themebuilder');
-const commands = require('devextreme-themebuilder/modules/commands');
-const baseParameters = require('devextreme-themebuilder/modules/base-parameters');
 const fs = require('fs');
 const path = require('path');
+const runCommand = require('../utility/run-command');
+const lock = require('../utility/file-lock');
+
+const commands = {
```
Comment (BingoRUS) [inline commands/themebuider.js]: why not read commands from `devextreme-themebuilder/modules/commands` after installing the package?

Comment (babich-a) [reply commands/themebuider.js]: Because of isThemeBuilderCommand method - if the command is for app creating we do not need to install the package

Comment (BingoRUS) [reply commands/themebuider.js]: Ah, you're right!

**Pull request merged**